package.json

```diff
@@ -15,8 +15,7 @@
     "start-pm2": "pm2 start ./server/server.js",
     "list-pm2": "pm2 list",
     "show-server-pm2": "pm2 show server",
-    "stop-pm2": "pm2 stop 0",
-    "seed-all": "npm-run-series seed seed seed seed seed seed seed seed seed seed"
```

Comment:
I assume this deletion is something you will want to keep?

Reply:
I think I accidentally deleted stop-pm2 but I deleted my seed-all script on purpose